tests/helpers.test.ts

```diff
@@ -116,12 +128,89 @@ describe('ModuleHelpers', () => {
     })
   })
 
-  describe('isCRATestCommand', () => {
+  describe('isCreateReactAppTestCommand', () => {
     it('should return true for CRA', () => {
-      expect(isCRATestCommand('react-scripts test --env=jsdom')).toBeTruthy()
+      expect(isCreateReactAppTestCommand('react-scripts test --env=jsdom')).toBe(true)
     })
+
     it('should return false for other scripts', () => {
-      expect(isCRATestCommand('custom-script')).toBeFalsy()
+      expect(isCreateReactAppTestCommand('custom-script')).toBe(false)
+    })
+  })
+
+  describe('pathToJest', () => {
+    const defaultSettings: any = {
+      pathToJest: null,
+      rootPath: '',
+    }
+
+    it('returns "npm test --" when bootstrapped with create-react-app', () => {
+      mockReadFileSync.mockReturnValueOnce(
+        JSON.stringify({
+          scripts: {
+            test: 'react-scripts test',
+          },
+        })
+      )
+
+      expect(pathToJest(defaultSettings)).toBe('npm test --')
+    })
+
+    it('returns the normalized "pathToJest" setting when set by the user', () => {
+      const expected = {}
```

Comment:
I find it more complicate to use an object and mock `normalize` than a custom string like `'customjest'`, but that's just my personal opinion and nothing which has to be changed.

Reply:
It's not great using the wrong type, but since it's an Object that's asserted with `expect().toBe()` you can be confident the value hasn't changed.